nri/style.css

```
@@ -0,0 +1,427 @@
+header {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  height: 60px;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  padding: 0 20px;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  background-color: white;
+  z-index: 777;
+}
+
+header h1 img {
+  width: 65px;
+}
+
+nav {
+  height: 100%;
+}
+
+.icons i {
+  font-size: 1.3rem;
+  margin-left: 25px;
+}
+
+.header-nav {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  list-style: none;
+  -webkit-box-pack: end;
+      -ms-flex-pack: end;
+          justify-content: flex-end;
+  padding-left: 200px;
+  height: 100%;
+  width: 100%;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+.header-nav a {
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+  font-size: 0.9rem;
+}
+.header-nav a:hover {
+  color: #0044ff;
+}
+.header-nav li {
+  margin-left: 20px;
+  height: 100%;
+  line-height: 60px;
+  white-space: nowrap;
+}
+.header-nav li:hover {
+  border-bottom: 2px solid black;
+}
+
+.box {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  margin-top: 70px;
+  padding-left: 60px;
+}
+.box p {
+  margin-left: 5px;
+  color: #999;
+  font-size: 0.7rem;
+}
+.box a {
+  text-decoration: none;
+  color: #999;
+}
+
+.big-bg {
+  background-image: url(./img/main.jpg);
+  width: 100%;
+  height: 250px;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  padding-top: 50px;
+  padding-left: 200px;
+  margin-top: 20px;
+}
+.big-bg p {
+  color: white;
+  font-size: 1.2rem;
+  margin-bottom: 20px;
+}
+.big-bg h2 {
+  color: white;
+  font-size: 1.8rem;
+}
+
+section {
+  width: 60%;
+  margin: 0 auto;
+}
+section h2 {
+  height: 70px;
+  line-height: 70px;
+  font-size: 1.5rem;
+}
+section h2 img {
+  width: 30px;
+  height: 30px;
+}
+section p {
+  margin-bottom: 35px;
+  line-height: 2rem;
+}
+section .section-img {
+  width: 100%;
+  margin: 0 auto;
+}
+section .section-img img {
+  width: 100%;
+}
+
+.trans-box {
+  text-align: center;
+  margin-top: 30px;
+}
+
+.trans {
+  background-color: #0c419a;
+  color: white;
+  text-decoration: none;
+  text-align: center;
+  display: inline-block !important;
```

`!important`は、後からスタイルの上書きができなくなり保守するときに大変な目に合うので使わないでください。
もし、!importantを使う明確な理由がある場合は、コードにコメントをしてください。

```suggestion
  display: inline-block;
```

（補足）
cssを書くときですが、基本的な考え方としては`クラス`だけを使ってスタイリングします。
（今回は別に対応する必要ないです）。

なぜかというと、クラスが一番スタイルの優先順位が低く、あとから追加変更しやすいからです。
タグ（`<h1>`とか）に直接スタイルを当てると、後からスタイルを追加しにくいです。

タグに当てるのは、最後仕方なくという感じが多いです。
（まぁ実際はプロジェクトの特徴や、納期、使い捨てなコードかどうかなど、いろいろな要因で変わりますが、基本的な考えとして知っておいてもらえたらいいと思います。）

---

なるほど。了解いたしました！